src/views/EdgeNode/FormFields/FormFieldsEdgeNode.vue

```diff
@@ -67,26 +51,16 @@
     <template #inputs>
       <div class="flex flex-col gap-5 mb-6">
         <div class="flex flex-col sm:max-w-lg w-full gap-2">
-          <label
-            for="name"
-            class="text-color text-base font-medium"
-            >Hash ID</label
-          >
-          <span class="p-input-icon-right">
-            <i class="pi pi-lock text-color-secondary" />
-            <InputText
-              class="w-full"
-              placeholder="Hash ID"
-              v-model="hashId"
-              id="hashId"
-              type="text"
-              :filled="true"
-              :disabled="true"
-            />
-          </span>
-          <small class="text-xs text-color-secondary font-normal leading-5">
-            The hash ID used to authenticate the edge node.
-          </small>
+          <FieldText
+            label="Hash ID"
+            description="The hash ID used to authenticate the edge node."
+            name="hashId"
+            :value="name"
+            :disabled="true"
```

Comment:
Caso suba algum ajuste recomendo isso:
```suggestion
            disabled
```